src/ConfigFactory.Core/IConfigModule.cs

```diff
@@ -48,10 +48,10 @@ public interface IConfigModule
     /// <returns>
     /// <see langword="true"/> if the validation was successful; <see langword="false"/> if the validation failed
     /// </returns>
-    public virtual bool Validate() => Validate(out _, out _);
+    public bool Validate() => Validate(out _, out _);
```

Comment:
No reason to hide this member

```suggestion
    public virtual bool Validate() => Validate(out _, out _);
```

Reply:
It is not hidden. Since it is in the interface it is by default virtual unless "sealed" or "private" keyword is used. So I just removed it for the sake of code's clarity. It is in no-way hidden method anyways. It remains virtual using the "virtual" keyword or not.
(see alo the explanation here: https://www.techpointfunda.com/2020/09/virtual-interface-method-csharp-8.html)

(But I have no problem leaving the keyword there if you prefer.)

Reply:
Cool, I didn't know that.

Either one works, I still find it clearer to leave the virtual keyword though.